stream.go

```diff
@@ -163,15 +169,54 @@ func (VSCodeObjectCodec) ReadObject(stream *bufio.Reader, v interface{}) error {
 	return json.NewDecoder(io.LimitReader(stream, int64(contentLength))).Decode(v)
 }
 
+// DEPRECATED: use NewPlainObjectStream
 // PlainObjectCodec reads/writes plain JSON-RPC 2.0 objects without a header.
```

Comment:
I believe the deprecation notice should be formatted as follows:

```
// PlainObjectCodec reads/writes plain JSON-RPC 2.0 objects without a header.
//
// Deprecated: Use NewPlainObjectStream
```

* [Godoc: documenting Go code](https://go.dev/blog/godoc)
* [Example in Go's standard lib](https://github.com/golang/go/blob/master/src/bytes/bytes.go#L755-L759)

Reply:
Fixed